manifests/0000_50_olm_15-packageserver.clusterserviceversion.yaml

```diff
@@ -9,7 +9,6 @@ spec:
   displayName: Package Server
   description: Represents an Operator package that is available from a given CatalogSource which will resolve to a ClusterServiceVersion.
   minKubeVersion: 1.11.0
-  replaces: packageserver.v0.9.0
```

Comment:
Why remove this? During an OpenShift upgrade, how will OLM know to clean up the `replaced` CSV, or to change ownership of the `APIService`?

Reply:
I updated the init to delete `packageserver.v0.9.0` and `packageserver.v0.10.0` - so whatever was there before, we're now moving to just having a single `packageserver` that we can update.

Reply:
are there clusters that still have 0.10.1?